### ISSUE TITLE
Improve select/deselect feature 

### ISSUE THREAD

**Issue by AnyoneMouse** [fast4x/RiMusic#1813]
### Description

I've noticed this before as well but didn't give much attention. In `Playlists` the select/deselect feature is pretty much useless. It's not much useful on `Queue` either.

The reasons:

1. As mentioned on #1408 **selection gets reset** if **scrolled outta** the view of **selected songs**. I've observed it more, let me explain it to you. The songs that aren't visible on sight anymore gets deselected. Suppose, you **selected 1, 2, 3** number songs **and 8th song as well** and you scrolled **outta the view of 1, 2 and 3** but **8 was still visible**. So, when you **scroll back up** while **keeping the 8th song visible** you'll see that **1-3 songs has been deselected**. Sorry if this explanation wasn't actually needed. 
2. After songs are selected **no quick access button appears in front**. For example `Delete`, `Download`.
3. **No _distinct_ button** visible **anywhere** to `download/remove/delete` **only** the **selected song/s.**
4. No `Add to playlist` button for selected song/s. When you try adding to playlist through the menu **all songs** get added to selected playlist **not just the selected songs**.
5. You can't `enqueue` selected songs. When you try using `enqueue` button all songs of playlist gets enqueued instead.

In short, this feature is _actually_ useless as of now.

### Suggested Solution

1. Ability to delete/remove/download etc. only selected songs when using the feature.
2. If possible show one or two quick access buttons when the select is in the process.
3. Fix the issue that has been mentioned on the point 1 on Description above.

**Comment by dAtA-TRoN** [comment]
https://github.com/fast4x/RiMusic/assets/71184627/a520aa32-d239-4729-8d1d-05e9d8bcdd05

**Comment by fast4x** [comment]
Fixed, available from version 0.6.38
